src/Spryker/Shared/RabbitMq/Transfer/rabbit_mq.transfer.xml

```diff
@@ -23,6 +23,7 @@
         <property name="queueOptionCollection" singular="queueOptionItem" type="RabbitMqOption[]"/>
         <property name="isDefaultConnection" type="bool"/>
         <property name="storeNames" type="array" singular="storeNames"/>
+        <property name="streamContextOptions" type="array" singular="streamContextOptions"/>
```

Comment:
as I know, for the new fields singular field, should be singular :)
We have it in plural for old cases to keep BC

Reply:
Uhh okay, I just used the same approach, I'll make it singular.